js/ui/scheduler/rendering_strategies/ui.scheduler.appointments.strategy.horizontal.js

```diff
@@ -14,7 +14,7 @@ var HorizontalRenderingStrategy = BaseAppointmentsStrategy.inherit({
     },
 
     calculateAppointmentWidth: function(appointment, position, isRecurring) {
-        var cellWidth = this._defaultWidth || this.getAppointmentMinSize(),
+        var cellWidth = this.getDefaultCellWidth() || this.getAppointmentMinSize(),
```

Comment:
I think it is a best practice to avoid addressing fields with underscores. Not only in children, but and in this class too. In other words, I just described a getter for a class field.